lib/pages/signup/pick-a-plan-page.js

```diff
@@ -83,4 +84,11 @@ export default class PickAPlanPage extends BaseContainer {
 		this.driver.wait( until.elementLocated( selector ), this.explicitWaitMS, 'Could not locate the business plan price element' );
 		return this.driver.findElement( selector ).getText();
 	}
+	static waitForPage( driver ) {
```

Comment:
Rather than override the BaseContainer function here I'd rather see it updated to take an optional parameter in case there are other places where we need to modify the default.  But I'm also fine just recording that in an issue labeled `task` and leaving this as-is if you don't have time to fix it now.

Reply:
> I'd rather see it updated to take an optional parameter in case there are other places where we need to modify the default.

Yeah good plan, I'll do that now.